cypress/support/fragments/circulation/circulation-rules.js

```diff
@@ -68,6 +71,13 @@ export default {
       cy.expect(circulationRules.rulesAsText).to.include(`n ${noticePolicyId}`);
     });
   },
+  createViaApi(newCirculatiuonPolicy) {
```

Comment:
may be good option to add return statement with policy attributes

Reply:
you mean to return exact params instead of body?

Reply:
Stop. I can't see where do you use this method?